### ISSUE TITLE
Change version in showTo

### ISSUE THREAD

**Issue by DanielAlves123** [CodeCrafter47/BungeeTabListPlus#204]
Hi, can you change versions to make it more simple? Instead of this: `  or ${viewer client_version} == "1.9"
  or ${viewer client_version} == "1.9.1"
  or ${viewer client_version} == "1.9.2"
  or ${viewer client_version} == "1.9.4"` it would be just `or ${viewer client_version} == "1.9`.

**Comment by CodeCrafter47** [comment]
The reason you need to have `1.9`, `1.9.1`, `1.9.2` and `1.9.4` separate is because these are not compatible, i.e. you can't join a `1.9.2` server with a `1.9` client.

On the other hand `1.11` and `1.11.2` are compatible which is, why BungeeTabListPlus only knows `1.11`, so it is sufficient to use `${viewer client_version} == "1.11"` to get both, `1.11` and `1.11.2` clients.

**Comment by DanielAlves123** [comment]
But you can make it more simple, right? And `${viewer client_version} == "1.11"` doesn't work with 1.11.2.

**Comment by CodeCrafter47** [comment]
Yeah, that was a bad example. But `1.11.1` and `1.11.2` are compatible and show up as `1.11.2`.

Also all `1.10` version are compatible to each other, and `1.8` to `1.8.9` show up as `1.8`.

The `client_version` placeholder has originally been introduced because people wanted to display it on the tab list. I'd consider it wrong if it were to display `1.9` even if it knows the player is using `1.9.4`. In cases where versions are compatible to each other it doesn't know which one a player is using, but in other cases it does.

Also you can do `${viewer client_version 3} == "1.9"` to check whether the version string starts with `1.9`.

**Comment by DanielAlves123** [comment]
${viewer client_version 3} == "1.11" doesn't work with 1.11.2

**Comment by CodeCrafter47** [comment]
You need to use `${viewer client_version 4} == "1.11"`, because you want to check whether the first 4 characters of the players version are `1.11`.
